### PR TITLE
Fix/prepared statement

### PR DESCRIPTION
**Pull request by IvanHdzC**
This PR changes PreparedStatement behaviour.
Before this PR the final statement was builded with native sql driver methods, but it's failing when inserting some SQL objects. The ones identified are Timestamp with TZ and functions.(An issue was created to take a deeper look at this miss behaviour #1959) 
With this PR the SQL query is builded as a plain String (as it does usually on the insert query).

### CODE REVIEW

**Review by AlvaroVega — APPROVED** [review]
LGTM

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackendImpl.java [hunk 1]
```diff
@@ -581,12 +581,21 @@ public void upsertTransaction (LinkedHashMap<String, ArrayList<JsonElement>> agg
             String insertQuery = SQLQueryUtils.sqlInsertQuery(aggregation,
```
Comment (fgalan) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackendImpl.java:581]: CNR entry (with reference to issue number, if we have some) should be included.

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackendImpl.java [hunk 2]
```diff
@@ -581,12 +581,21 @@ public void upsertTransaction (LinkedHashMap<String, ArrayList<JsonElement>> agg
...
-                    destination).toString();
+                    destination,
+                    attrNativeTypes).toString();
 
-            PreparedStatement insertStatement = null;
+            PreparedStatement insertStatement;
             insertStatement = connection.prepareStatement(insertQuery);
+            /*
+
+            FIXME
```
Comment (fgalan) [inline cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackendImpl.java:591]: Maybe this FIXME should mention an issue number? Is this related with #1959?

Comment (fgalan) [reply cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackendImpl.java:591]: (Same in the other FIXME below)

**Review by manucarrace — APPROVED** [review]
LGTM

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackendImpl.java (hunk 1 above)
Comment (IvanHdzC) [reply cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackendImpl.java:582]: Fixed in [2f9140f](https://github.com/telefonicaid/fiware-cygnus/pull/1960/commits/2f9140fe589e08bee19e85ea1daeca198e1e0f97)

File: cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackendImpl.java (hunk 2 above)
Comment (IvanHdzC) [reply cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/sql/SQLBackendImpl.java]: Fixed in (6405c46)[https://github.com/telefonicaid/fiware-cygnus/pull/1960/commits/6405c4618697b38182c45733d38268d817e903fd]

**Review by fgalan — APPROVED** [review]
LGTM
